app/src/main/res/values/strings.xml

```diff
@@ -17,7 +17,6 @@
   <string name="save_media_error">An error occurred when trying to save the media!</string>
   <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
```

Comment:
This is the only strings file that should be edited.

Reply:
Ok. Thanks! I no know why this build failing..